src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeFetchers.scala

```diff
@@ -60,19 +63,20 @@ trait BlocksDataFetchers {
 
     def makeUrl = (offset: Offset) => s"blocks/${hashRef.value}~${String.valueOf(offset)}"
 
+    // lazy val failureHandler = (offset: Offset, error: Throwable) =>
```

Comment:
How come these failure handlers are commented out? How are HTTP errors being handled now?

Reply:
In this version they're temporarily set off, yet this branch was never meant to get merged and deployed